### ISSUE TITLE
Release 4.9.0 - Alpha 2 - Footprint metrics

### ISSUE THREAD

**Issue by wazuhci** [wazuh/wazuh#24574]
### Footprint metrics information

|||
| :--                              | :--                                                             |
| **Main release stage issue**     | #24573                                         |
| **Version**                      | 4.9.0                                                   |
| **Release stage**                | Alpha 2                                                   |
| **Tag**                          | https://github.com/wazuh/wazuh/tree/v4.9.0-alpha2          |
| **Previous footprint metrics**   | #24255                                              |

---

| Test | Url |
| -    | -   |
| 2.5d  | https://ci.wazuh.info/job/Test_stress_tier/176/ |

---

## Checks

Status | Modules                                         | 4.9.0 Issue (2.5d)  |
| :--: | :--                                             | :--:                        |
| :yellow_circle:   | All                                             | https://github.com/wazuh/wazuh/issues/24598                         |
| :yellow_circle:   | Vulnerability-detector-Register                 | https://github.com/wazuh/wazuh/issues/24605                         |
| :yellow_circle:   | Logcollector                                    | https://github.com/wazuh/wazuh/issues/24610                         |
| :yellow_circle:   | Syscheck                                        | https://github.com/wazuh/wazuh/issues/24597                         |
| :yellow_circle:   | Rootcheck                                       | https://github.com/wazuh/wazuh/issues/24600                         |
| :yellow_circle:   | SCA                                             | https://github.com/wazuh/wazuh/issues/24611                         |
| :yellow_circle:   | Active Response                                 | https://github.com/wazuh/wazuh/issues/24603                         |
| :yellow_circle:   | Syscollector                                    | https://github.com/wazuh/wazuh/issues/24602                         |
| :red_circle:   | Docker,Ciscat,Osquery,Azure,Openscap            | https://github.com/wazuh/wazuh/issues/24574#issuecomment-2229320318                         |
| :yellow_circle:   | All-except-Logcollector                         | https://github.com/wazuh/wazuh/issues/24608                         |
| :yellow_circle:   | All-except-Syscheck                             | https://github.com/wazuh/wazuh/issues/24609                         |
| :yellow_circle:   | All-except-Rootcheck                            | https://github.com/wazuh/wazuh/issues/24612                         |
| :yellow_circle:   | All-except-SCA                                  | https://github.com/wazuh/wazuh/issues/24606                         |
| :yellow_circle:   | All-except-Active Response                      | https://github.com/wazuh/wazuh/issues/24604                         |
| :yellow_circle:   | All-except-Syscollector                         | https://github.com/wazuh/wazuh/issues/24599                         |
| :yellow_circle:   | All-except-Docker,Ciscat,Osquery,Azure,Openscap | https://github.com/wazuh/wazuh/issues/24613                         |
| :yellow_circle:   | Logcollector-Syscollector                       | https://github.com/wazuh/wazuh/issues/24607                         |
| :yellow_circle:   | Logcollector-SCA                                | https://github.com/wazuh/wazuh/issues/24601                         |
| :red_circle:    | Logcollector-Syscheck                           | https://github.com/wazuh/wazuh/issues/24574#issuecomment-2229320318                         |
| :yellow_circle:   | macOS-All                                       | https://github.com/wazuh/wazuh/issues/24614                         |
| :yellow_circle:   | Solaris-All                                     | https://github.com/wazuh/wazuh/issues/24614                         |

---

Status legend:
:black_circle: - Not started/In progress
:white_circle: - Skipped
:red_circle: - Found new errors/bugs/failures
:yellow_circle: - Found known errors/bugs/failures
:green_circle: - All tests passed

---

## Auditor's validation

In order to close and proceed with the release or the next candidate version, the following auditors must give the green light to this RC.

- [ ] @davidjiglesias

---

**Comment by fcaffieri** [comment]
### Status failed test

The `Docker,Ciscat,Osquery,Azure,Openscap` and `Logcollector-Syscheck` tests failed due to the error reported in this [issue](https://github.com/wazuh/wazuh-jenkins/issues/5478)

**Comment by rauldpm** [comment]
@fcaffieri @pcapellan For this stage we will only check plots, we want to check if fragmentation and leaks appear, we will compare with 4.8.0 so the next one will be 4.8.1, where we will do a complete search

- Note: Report big changes in plots, ignore small usage changes (unless behavior changes)

**Comment by fcaffieri** [comment]
Closing this issue because there was a problem with parcels. The 4.9.0 Alpha 2 packages were overwritten with those of 4.9.0 Alpha 1 when the release of 4.8.1 RC 3 was made. That said, the tests are meaningless because the wrong package was used.
Related issue: https://github.com/wazuh/wazuh-jenkins/issues/6747
The closing of this issue was authorized by @rauldpm

**Comment by rauldpm** [comment]
## About missing tests

- DOCKER-LISTENER,CIS-CAT,OSQUERY,AZURE-LOGS
  - https://ci.wazuh.info/job/Test_stress/5418/consoleFull


```
Error Spot Task Definition was unilaterally stopped by AWS.
```

- LOGCOLLECTOR,SYSCHECK
  - https://ci.wazuh.info/job/Test_stress/5417/consoleFull

```
aws --region xxx ecs run-task --cluster Jenkins --count 1 --task-definition graphic_generation_4-9-0:15 --cli-input-json [file://ECS_run_B5417_TIDxxx_monitor-agent-Test_stress_B5417_centos-pre-release_xxx.json](file:///)
17:12:03  [Pipeline] readJSON
17:12:03  [Pipeline] echo
17:12:03  No task data provided, skipping setting task data.
17:12:03  [Pipeline] sh
17:12:03  + aws --region us-east-1 ecs describe-tasks --cluster Jenkins --tasks null
17:12:05  [Pipeline] sh
17:12:05  + cat stderr.txt
17:12:05  [Pipeline] echo
17:12:05  java.lang.Exception: hudson.AbortException: script returned exit code 254
17:12:05  An error occurred (InvalidParameterException) when calling the DescribeTasks operation: taskId length should be one of [32,36]
17:12:05
```

It seems to be a bug with the `Logcollector,syscheck`, it is weird that this happened only once, we need to investigate it, I have relaunched the missing test, and we will link them to have them as references, @fcaffieri please open an issue about this error

**Comment by fcaffieri** [comment]
Update [comment](https://github.com/wazuh/wazuh/issues/24574#issuecomment-2229320318) with new issue
